assets/wire/components/RenditionData.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ArticlePicture from 'ui/components/ArticlePicture';
+import ArticleMedia from 'ui/components/ArticleMedia';
+import {isCustomRendition, isKilled} from 'wire/utils';
+import {IArticle} from 'interfaces';
+
+interface IProps {
+  data: IArticle;
```

Comment:
I was rather concerned about the param name, not the component. so this should be rather `association`